### DISCUSSION TITLE
Debugger terminology: "class variables"

### DISCUSSION THREAD

**Discussion by pamelafox** [microsoft/debugpy#1305] [Q&A]
This isn't really a bug, but I just wanted to share my feedback that I'm a bit thrown off by the terminology used in the Debugger panel when I've been debugging recently. It shows "special variables" and "class variables" under "Locals".

<img width="955" alt="Screenshot 2023-05-30 at 10 26 39 AM" src="https://github.com/microsoft/vscode-python/assets/297042/8e30f247-0fb1-42da-9dda-f0b99cb3c76e">

I believe "special variables" refers to either Python-designated special attributes (as described in https://docs.python.org/3/reference/datamodel.html) or maybe it just lists all dunder-named attributes (including user-defined ones), I haven't figured that out yet.

When I see "class variables", I assume it refers to class attributes, i.e. attributes that belong to the entire class versus each instance (also described in that doc above). However, it seems instead to refer to any variable that is an instance of a class?

<img width="398" alt="Screenshot 2023-05-30 at 10 36 42 AM" src="https://github.com/microsoft/vscode-python/assets/297042/3ffdf48c-8a6b-4e34-86e1-f904396abec7">

In the example above, none of those are what I consider to be class variables.

This terminology is confusing to me, and keeps throwing me off while debugging.

**Comment by brettcannon** [comment]
I believe this is all coming from debugpy, so I think they are the ones that would need to make a change.

**Reply by pamelafox** [reply]
Ah I see, shall I move this discussion to https://github.com/microsoft/debugpy/discussions ?
